shinano-common-caf/proprietary/Android.mk

```diff
@@ -51,7 +51,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE_TAGS   := eng
 LOCAL_MODULE        := wait4tad_static
 LOCAL_MODULE_CLASS  := BIN
-LOCAL_SRC_FILES     := sbin/wait4tad_static
+LOCAL_SRC_FILES     := vendor/sbin/wait4tad_static
```

Comment:
Less important here to keep it into ramdisk but should be good for consistency to keep it into ramdisk too.